### ISSUE TITLE
npm installation broken?

### ISSUE THREAD

**Issue by madeofpeople** [superwolff/metalsmith-in-place#14]
```
$ npm install --save metalsmith-in-place
metalsmith-in-place@1.1.0 node_modules/metalsmith-in-place
├── extend@1.2.1
├── debug@0.7.4
├── consolidate@0.10.0
├── async@0.2.10
├── multimatch@0.1.0 (lodash@2.4.2, minimatch@0.2.14)
└── lodash.omit@2.4.1 (lodash._baseflatten@2.4.1, lodash._basedifference@2.4.1, lodash.createcallback@2.4.3, lodash.forin@2.4.1)
$ metalsmith

  Metalsmith · failed to require plugin "metalsmith-in-place".

$
```
going into the node_modules/metalsmith-in-place/ directory and doing npm install manually, fixes this issue. 

The tests also fail until I do a manual install inside the module directory.

**Comment by ismay** [comment]
Hmm, were there any extra dependencies installed when you did the second `npm install`? And what were the errors when the tests were failing?

**Comment by madeofpeople** [comment]
Here's a super simplified example

package.json
```
{
  "name": "test",
  "private": true,
  "dependencies": {
    "metalsmith": "~1.7.0",
    "metalsmith-in-place": "^1.1.0",
    "metalsmith-layouts": "^1.1.0",
    "metalsmith-debug": "*"
  },
  "repository": {
    "type": "git",
    "url": ""
  }
}

```

metalsmith.json
```
{
  "plugins": {
    "metalsmith-debug": {},
    "metalsmith-in-place": {
      "engine": "swig"
    }
  }
}
```

cli
```
$ rm -rf node_modules/
$ npm install
metalsmith-debug@0.0.2 node_modules/metalsmith-debug
└── debug@1.0.4 (ms@0.6.2)

metalsmith-layouts@1.1.0 node_modules/metalsmith-layouts
├── extend@3.0.0
├── async@1.3.0
├── debug@2.2.0 (ms@0.7.1)
├── lodash.omit@3.1.0 (lodash._arraymap@3.0.0, lodash._bindcallback@3.0.1, lodash._pickbyarray@3.0.2, lodash.restparam@3.6.1, lodash._pickbycallback@3.0.0, lodash._baseflatten@3.1.4, lodash.keysin@3.0.8, lodash._basedifference@3.0.3)
├── multimatch@2.0.0 (array-differ@1.0.0, array-union@1.0.1, minimatch@2.0.9)
└── consolidate@0.13.1 (bluebird@2.9.34)

metalsmith-in-place@1.1.0 node_modules/metalsmith-in-place
├── extend@1.2.1
├── debug@0.7.4
├── consolidate@0.10.0
├── async@0.2.10
├── lodash.omit@2.4.1 (lodash._baseflatten@2.4.1, lodash._basedifference@2.4.1, lodash.forin@2.4.1, lodash.createcallback@2.4.3)
└── multimatch@0.1.0 (minimatch@0.2.14, lodash@2.4.2)

metalsmith@1.7.0 node_modules/metalsmith
├── absolute@0.0.1
├── is-utf8@0.2.0
├── thunkify@2.1.2
├── clone@0.1.19
├── stat-mode@0.2.1
├── is@2.2.1
├── unyield@0.0.1 (co@3.1.0)
├── async@0.9.2
├── commander@2.8.1 (graceful-readlink@1.0.1)
├── ware@1.3.0 (wrap-fn@0.1.4)
├── chalk@0.5.1 (ansi-styles@1.1.0, escape-string-regexp@1.0.3, supports-color@0.2.0, strip-ansi@0.3.0, has-ansi@0.1.0)
├── gray-matter@2.0.1 (extend-shallow@2.0.1, ansi-red@0.1.1)
├── recursive-readdir@1.2.1 (minimatch@0.3.0)
├── fs-extra@0.10.0 (jsonfile@1.2.0, ncp@0.5.1, mkdirp@0.5.1)
├── co-fs-extra@0.0.2 (co-fs@1.2.0, fs-extra@0.12.0)
├── rimraf@2.4.2 (glob@5.0.14)
└── gnode@0.1.1 (regenerator@0.8.34)
$ metalsmith

  Metalsmith · failed to require plugin "metalsmith-in-place".

$ cd node_modules/metalsmith-in-place/
$ npm install
is-utf8@0.2.0 node_modules/is-utf8

assert-dir-equal@0.0.1 node_modules/assert-dir-equal
└── fs-readdir-recursive@0.0.1

mocha@1.21.5 node_modules/mocha
├── escape-string-regexp@1.0.2
├── diff@1.0.8
├── growl@1.8.1
├── commander@2.3.0
├── debug@2.0.0 (ms@0.6.2)
├── mkdirp@0.5.0 (minimist@0.0.8)
├── glob@3.2.3 (inherits@2.0.1, graceful-fs@2.0.3, minimatch@0.2.14)
└── jade@0.26.3 (commander@0.6.1, mkdirp@0.3.0)

swig@1.3.2 node_modules/swig
├── optimist@0.6.1 (wordwrap@0.0.3, minimist@0.0.10)
└── uglify-js@2.4.23 (uglify-to-browserify@1.0.2, yargs@3.5.4, source-map@0.1.34)

metalsmith@1.7.0 node_modules/metalsmith
├── absolute@0.0.1
├── thunkify@2.1.2
├── clone@0.1.19
├── stat-mode@0.2.1
├── is@2.2.1
├── unyield@0.0.1 (co@3.1.0)
├── async@0.9.2
├── commander@2.8.1 (graceful-readlink@1.0.1)
├── ware@1.3.0 (wrap-fn@0.1.4)
├── chalk@0.5.1 (ansi-styles@1.1.0, escape-string-regexp@1.0.3, supports-color@0.2.0, has-ansi@0.1.0, strip-ansi@0.3.0)
├── gray-matter@2.0.1 (ansi-red@0.1.1, extend-shallow@2.0.1)
├── fs-extra@0.10.0 (jsonfile@1.2.0, ncp@0.5.1, mkdirp@0.5.1)
├── recursive-readdir@1.2.1 (minimatch@0.3.0)
├── co-fs-extra@0.0.2 (co-fs@1.2.0, fs-extra@0.12.0)
├── rimraf@2.4.2 (glob@5.0.14)
└── gnode@0.1.1 (regenerator@0.8.34)
$ cd ../../
$ metalsmith

  Metalsmith · successfully built to /Users/thiago/repos/kalastatic/build

$ 
```

**Comment by ismay** [comment]
Thanks for the details, should be fixed now in 1.1.1. Could you check whether it's solved now?

**Comment by ismay** [comment]
Fixed in f05d33b9814af5b0b941e66860a174ae6c40df5d
